roles/taca/tasks/main.yml

```diff
@@ -0,0 +1,38 @@
+- name: Fetch TACA-ngi-pipeline from GitHub
+  git: repo={{ taca_ngi_repo }}
+       dest={{ taca_ngi_dest }}
+       version={{ taca_ngi_version }}
+       force=yes
+  tags: taca
+
+- name: Install TACA-ngi-pipeline requirements
+  shell: "{{ ngi_pipeline_venv }}/bin/pip install" -r "{{ taca_ngi_dest }}/requirements.txt"
+  tags: taca
+
+- name: Install TACA-ngi-pipeline
+  shell: cd "{{ taca_ngi_dest }}" && "{{ ngi_pipeline_venv }}/bin/pip" install .
+  tags: taca
+
+-name: Create TACA delivery configs directory
+  file: path="{{ ngi_resources }}/TACA" state=directory mode=g+s
+  tags: taca
```

Comment:
`file` and `tags` shouldn't be indented. I guess it would make more sense to deploy the configs under `{{ ngi_pipeline_conf }}` as well, right?